src/main/java/imagejan/plugins/BlindFiles.java

```diff
@@ -0,0 +1,105 @@
+package imagejan.plugins;
```

Comment:
Let's stick with the `sc.fiji` package, and move the repository to the fiji org as discussed [on the forum](https://forum.image.sc/t/blind-experiment-is-this-packaged-on-an-update-site/19142/9?u=imagejan).

Reply:
Ok, I will refactor that.